apps/dashboard/src/app/(main)/(protected)/projects/[projectId]/metrics/page-client.tsx

```diff
@@ -86,32 +33,87 @@ const dauConfig = {
       {
         data !== null && <>
-          <h2>Key Metrics</h2>
-          <LineChartDisplay
-            config={totalUsersConfig}
-            datapoints={data.total_users}
-          />
-          <LineChartDisplay
-            config={dauConfig}
-            datapoints={data.daily_active_users}
-          />
+          <GlobeSection countryData={data.users_by_country}>
+            <Card>
+              <CardContent>
+                <CardTitle className='text-2xl'>
+                  {data.total_users}
+                </CardTitle>
+                <CardDescription className='text-xl'>Total Users</CardDescription>
+              </CardContent>
+              <CardContent>
+                <CardTitle className='text-2xl'>
+                  {data.daily_active_users[data.daily_active_users.length - 1].activity}
+                </CardTitle>
+                <CardDescription className='text-xl'>Active Users Today</CardDescription>
+              </CardContent>
+            </Card>
+          </GlobeSection>
+          <div className='grid gap-4 lg:grid-cols-2'>
+            <LineChartDisplay
+              config={dailyRegistrationsConfig}
+              datapoints={data.daily_users}
+            />
+            <LineChartDisplay
+              config={dauConfig}
+              datapoints={data.daily_active_users}
+            />
+            <Card>
+              <CardHeader>
+                <CardTitle>Latest Registrations</CardTitle>
+              </CardHeader>
+              <CardContent>
+                <Table>
+                  <TableHeader>
+                    <TableRow>
+                      <TableHead>Display Name</TableHead>
+                      <TableHead>Registered at</TableHead>
+                    </TableRow>
+                  </TableHeader>
+                  <TableBody>
+                    {
+                data.recently_registered.map((user: any) => <TableRow key={user.id}>
+                  <TableCell>{user.display_name}</TableCell>
+                  <TableCell>{fromNow(new Date(user.created_at))}</TableCell>
+                </TableRow>)
+                    }
+                  </TableBody>
+                </Table>
+              </CardContent>
+            </Card>
+            <Card>
+              <CardHeader>
+                <CardTitle>Recently Active Users</CardTitle>
+              </CardHeader>
+              <CardContent>
+                <Table>
+                  <TableHeader>
+                    <TableRow>
+                      <TableHead>Display Name</TableHead>
+                      <TableHead>Registered at</TableHead>
+                    </TableRow>
+                  </TableHeader>
+                  <TableBody>
+                    {
+                data.recently_active.map((user: any) => <TableRow key={user.id}>
+                  <TableCell>{user.display_name}</TableCell>
+                  <TableCell>{fromNow(new Date(user.updated_at))}</TableCell>
```

Comment:
updated_at is not the same as last_active_at. This should be last_active_at (calculated by finding the most recent UserActivityEvent)